guides/policy/azure-authenticator/index.html

```diff
@@ -0,0 +1,140 @@
+<h1>Azure Authenticator Policy Generator</h1>
+
+Generate Host Policy needed to use Azure Authenticator using Azure token
```

Comment:
Generate a policy defining a host that can authenticate with the Azure Authenticator, using an Azure AD token